### ISSUE TITLE
Problem using May 2021 vcpkg release

### ISSUE THREAD

**Issue by 0-wiz-0** [lukka/run-vcpkg#85]
We've set up github actions to build libzip on Windows, and use run-vcpkg to install the necessary dependencies.
We initially based our setup on what scummvm does, since the documentation in the Quickstart section of the README doesn't even mention that vcpkgGitCommitId is required.

scummvm is using the November 2020 release of vcpkg so we started with that and got it working:
https://github.com/nih-at/libzip/blob/master/.github/workflows/build.yml
https://github.com/nih-at/libzip/runs/2859455515?check_suite_focus=true

but there was an issue in liblzma, so I tried switching to the May 2021 release of vcpkg:
https://github.com/nih-at/libzip/commit/6405b8dfde23b7d6a7a8b31e2cbb8723b5ec7e1c#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721

However, after that change it does not finish building bzip2 any longer (though the bzip2 package itself does not have any changes between the two tags):
https://github.com/nih-at/libzip/runs/2859844478?check_suite_focus=true

I'm not sure if you can see the logs, so here's part of the log:
```
Install/Update ports
  Running 'vcpkg remove --outdated --recurse ' in directory 'D:\a\libzip\libzip\vcpkg' ...
  There are no outdated packages.
  Using triplet 'x64-windows'.
  Set the environment variable 'RUNVCPKG_VCPKG_TRIPLET' to value: x64-windows
  Set the output variable 'RUNVCPKG_VCPKG_TRIPLET_OUT' to value: x64-windows
  Running 'vcpkg install --recurse zlib bzip2 liblzma zstd --triplet x64-windows --clean-after-build' in directory 'D:\a\libzip\libzip\vcpkg' ...
  Computing installation plan...
  The following packages will be built and installed:
      bzip2[core]:x64-windows -> 1.0.8#1
      liblzma[core]:x64-windows -> 5.2.5#2
      zlib[core]:x64-windows -> 1.2.11#10
      zstd[core]:x64-windows -> 1.4.9
  Detecting compiler hash for triplet x64-windows...
  Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\43\43b505470d324322516cdf2f8f95df303ec4da8e.zip
  Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\19\19e7a8b72f5b408a7a16139961f88f4430ebd788.zip
  Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\3d\3dba13f76ad5a5bd7b46fffcb031c1ed46d375be.zip
  Could not locate cached archive: C:\Users\runneradmin\AppData\Local\vcpkg\archives\b8\b8ff9a9ecba3ccb5ada59896c90cdaab1a1d472d.zip
  Starting package 1/4: bzip2:x64-windows
  Building package bzip2[core]:x64-windows...
  -- Fetching https://sourceware.org/git/bzip2.git...
  CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:91 (file):
    file failed to open for reading (No such file or directory):
  
      D:/a/libzip/libzip/vcpkg/buildtrees/bzip2/git-init-x64-windows-out.log
  Call Stack (most recent call first):
    scripts/cmake/vcpkg_from_git.cmake:85 (vcpkg_execute_required_process)
    ports/bzip2/portfile.cmake:1 (vcpkg_from_git)
    scripts/ports.cmake:142 (include)
  
  
  Error: Building package bzip2:x64-windows failed with: BUILD_FAILED
  Please ensure you're using the latest portfiles with `.\vcpkg update`, then
  submit an issue at https://github.com/Microsoft/vcpkg/issues including:
    Package: bzip2:x64-windows
    Vcpkg version: 2021-05-05-9f849c4c43e50d1b16186ae76681c27b0c1be9d9
  
  Additionally, attach any relevant sections from the log files above.
```

I then tried adding a manual 'vcpkg update' step and some other things, but couldn't get it to work.
(see https://github.com/nih-at/libzip/commits/master/.github/workflows/build.yml for details)

Is this a problem in vcpkg-run or in vcpkg? I'm not sure where to report this.
Thank you.

**Comment by lukka** [comment]
@0-wiz-0 this should be a `vcpkg` problem, but I'd would try first using the most recent version of the `run-vcpkg@v7` which should be able to attach the log files (if any) to result of the build (i.e. https://github.com/lukka/run-vcpkg/issues/83). Those files may reveal an insight of what the problem is.

**Comment by 0-wiz-0** [comment]
Using v7 didn't help.
I've reported this at vcpkg, let's see what they find. Thanks!

**Comment by lukka** [comment]
@0-wiz-0 
By inspecting your [action executions](https://github.com/nih-at/libzip/actions) the problem was solved by using `vcpkg` at commit id `0bf3923f9fab4001c00f0f429682a0853b5749e0`, rather than the newer `5568f110b509a9fd90711978a7cb76bae75bb092`.
Closing the issue as it is not related to `run-vcpkg`.
